### PR TITLE
hotfix-145 Added UTM Checks

### LINKED ISSUE #145: Add UTM and local storage parms
Similar to current, where there are parameters that set local storage and then are picked up by the datalayer and Marketo.

### PR DESCRIPTION
**Pull request by davenichols-DHLS**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #145

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://hotfix-145-local-storage-parms--danaher-ls-aem--hlxsites.hlx.page/

### CODE REVIEW

File: scripts/delayed.js
Comment (mhaack) [inline scripts/delayed.js:1]: I think we can move that to scripts.js and run it in the lazy phase (via `loadLazy`)

Comment (mhaack) [reply scripts/delayed.js:1]: I suggest we move this to scripts.js into its own function and call it at the end of the lazy phase, after https://github.com/hlxsites/danaher-ls-aem/blob/f0db1ea9e78a4d0ee2d57381c36948e7523f67b1/scripts/scripts.js#L116

File: scripts/scripts.js [hunk 1]
```diff
@@ -93,6 +93,49 @@ async function loadEager(doc) {
   }
 }
 
+// UTM Paramaters check - start
+function getParameterByName(parameter, url = window.location.href) {
```
Comment (duynguyen) [inline scripts/scripts.js:97]: @davenichols-DHLS there are a few lint errors in the new code. They might need to be fixed or you can add the lint ignore comment so the test won't complain anymore.

Comment (davenichols-DHLS) [reply scripts/scripts.js:97]: @rgravitvl , @amlan18041996  and @duynguyen - I have tried to remove the lint errors and rewritten the code but it is still not happy. Can you please help advise or confirm if I can just ask lint to ignore this whole script?

**Review by duynguyen — APPROVED** [review]

**Pull request merged**